specification/maps/data-plane/Creator/preview/2022-09-01-preview/style.json

```diff
@@ -102,21 +102,28 @@
   "paths": {
     "/styles": {
       "post": {
-        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nCreator makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. [This](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps) article introduces concepts and tools that apply to Azure Maps Creator.\n\n The Style Import API allows the caller to import an azure maps style.\n\n## Submit Import Request\n\nTo import your style you will make a `POST` request with an azure maps style. \n\nThe Import Style API is a [long-running request](https://aka.ms/am-creator-lrt-v2).",
-        "operationId": "Style_Import",
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe Style Create request allows the caller to create an Azure Maps style.\n\nCreator makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. [This](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps) article introduces concepts and tools that apply to Azure Maps Creator.\n\n## Submit Create Request\n\nTo create your style you will make a `POST` request with an azure maps style. \n\nThe Create Style API is a [long-running request](https://aka.ms/am-creator-lrt-v2).",
+        "operationId": "Style_Create",
         "x-ms-long-running-operation": true,
         "x-ms-long-running-operation-options": {
           "final-state-via": "location"
         },
         "x-ms-examples": {
-          "Import Style by providing an azure maps style json file": {
-            "$ref": "./examples/style/Style_Import.json"
+          "Create Style by providing an azure maps style json file": {
+            "$ref": "./examples/style/Style_Create.json"
           }
         },
         "parameters": [
           {
             "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
           },
+          {
+            "name": "Content-Type",
+            "description": "Specify the request body content type. `application/zip` if request is a zip file, `application/json` if request body is a JSON file.",
+            "type": "string",
+            "in": "header",
+            "required": true
+          },
```

Comment:
Don't specify `Content-Type` explicitly -- use `consumes`.

Reply:
Switched to use consumes for Content-Type